SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -164,7 +135,14 @@ void USpatialGameInstance::TryConnectToSpatial()
 
 void USpatialGameInstance::StartGameInstance()
 {
-	TryConnectToSpatial();
+	if (GetDefault<USpatialGDKSettings>()->GetPreventClientCloudDeploymentAutoConnect())
```

Comment:
Shouldn't this be wrapped in a `if server` or `#IF WITH_EDITOR`? 

I ask because I sometimes use commandline args to boot my clients and connect to a deploy. I wouldn't want `GetPreventClientCloudDeploymentAutoConnect` which is cloud only to block that.

Then again knowing if you're connecting to a cloud or local deploy is not a solved issue.

Reply:
Also when UE starts external process clients and servers aren't they using commandline args?

Reply:
Ah it is commandline args but it's only a URL. Sorry.
We perhaps should make the distinction between URL commandline and SpatialOS config commandline in the future.